better_exceptions_hook/__init__.py,sha256=pA0qhpouUIuUqfeSqNNq8XOUcmxLPDMWGH-mv2SbpHM,123
better_exceptions_hook-1.0.0.data/data/lib/python3.6/site-packages/better_exceptions_hook.pth,sha256=qhUrpdIrdw_0ye8BBo7bx0d5mkjiHMAxc-XLgXW3ypQ,61
better_exceptions_hook-1.0.0.dist-info/DESCRIPTION.rst,sha256=OCTuuN6LcWulhHS3d5rfjdsQtW22n7HENFRh6jC6ego,10
better_exceptions_hook-1.0.0.dist-info/METADATA,sha256=0g_2DypH0lEC620ZFA-_GGragDiI0hANocVRyzdWXqQ,310
better_exceptions_hook-1.0.0.dist-info/RECORD,,
better_exceptions_hook-1.0.0.dist-info/WHEEL,sha256=rNo05PbNqwnXiIHFsYm0m22u4Zm6YJtugFG2THx4w3g,92
better_exceptions_hook-1.0.0.dist-info/metadata.json,sha256=0ja3UVZeRfj2FeGi8Tn2NUXcRm48AMeoXxIe7kQhECc,545
better_exceptions_hook-1.0.0.dist-info/top_level.txt,sha256=_ITyVcIPuAOfcbcQ_hggjbcLV7Mey2k5oXUYNSRPVGg,23
